gnomad_qc/v3/resources/constants.py

```diff
@@ -1,8 +1,23 @@
-RELEASES = ["3", "3.1", "3.1.1"]
+RELEASES = ["3.0", "3.1", "3.1.1"]
 CURRENT_RELEASE = "3.1.1"
+
 HGDP_TGP_RELEASES = ["3.1"]
 CURRENT_HGDP_TGP_RELEASE = "3.1"
-CURRENT_META_VERSION = "3.1"
+
 META_VERSIONS = ["3", "3.1"]
-CURRENT_PROJECT_META_VERSION = "3.1"
+CURRENT_META_VERSION = "3.1"
+
 PROJECT_META_VERSIONS = ["3", "3.1"]
+CURRENT_PROJECT_META_VERSION = "3.1"
+
+ANNOTATION_VERSIONS = ["3", "3.1"]
```

Comment:
It would probably be worth adding some comments here about why there are so many different version lists when they're all very similar.

Reply:
Would it be better if I made something like a `DEFAULT_VERSIONS = ["3", "3.1", "3.1.1"]` and then removed where needed with added comments?

Reply:
I think the way it is now is possibly easier to read and understand than adding/removing from other version lists.

Also, maybe the issue is just that I'm not familiar with the different pieces that go into making a gnomAD release. If it's obvious what these are for someone who is familiar, feel free to ignore my comment.

Reply:
No I think this is a good comment, I changed it up a little to simplify it, let me know if it looks better